contrib/lang/asciidoc/keybindings.el

```diff
@@ -0,0 +1,34 @@
+;;; keybindings.el --- Asciidoc Layer key-bindings File
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2015 Mark Safronov, Torben Hoffmann & Contributors
+;;
+;; Author: Mark Safronov <hijarian@gmail.com>
+;; Author: Torben Hoffmann <torben.lehoff@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;; We have quite a lot of possible keybindings. See `adoc-mode.el', its bottom part where the huge easy-menu is defined and after that, where the various `tempo-template-*' functions are defined.
+
+;;; See /doc/CONVENTIONS.md#plain-text-markup-languages
+(evil-leader/set-key-for-mode 'adoc-mode
```

Comment:
This should go in the `use-package` block in `packages.el`, `keybindings.el` is for bindings not associated with an ELPA package.